tensorflow_blade/build.py

```diff
@@ -172,6 +173,9 @@ def _write(line, cmd="build"):
         _opt("cxxopt", "-DBUILD_WITH_BAZEL")
         _action_env("BUILD_WITH_BAZEL", "1")
         _action_env("PYTHON_BIN_PATH", which("python3"))
+        _action_env("GCC_HOST_COMPILER_PATH", which("gcc"))
```

Comment:
why it's moved to common part?

Reply:
I think this needed for all devices build, thus move to `common` part.